### PR TITLE
[WIP] test(workflow): fix yarn tests

### LINKED ISSUE #57: Fix failing tests
It looks like the github workflow requires installation of all the package managers and using package managers to build applications. Add installation of package managers (eg: yarn) and building of test package (yarn install in the right location).

### PR DESCRIPTION
**Pull request by ba11b0y**
Fixes #57

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -17,5 +17,14 @@ jobs:
         with:
           go-version: 1.19
           check-latest: true
+      - name: Setup node
+        uses: actions/setup-node@v3
```
Comment (cpanato) [inline .github/workflows/test.yml:21]: can you please use the git hash for the latest release? similar we have for the others, thanks!

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -17,5 +17,14 @@ jobs:
...
+          with:
+            node-version: 18.16.0
+      - name: Install dependencies with yarn
+        uses: borales/actions-yarn@v4
```
Comment (cpanato) [inline .github/workflows/test.yml:25]: same here

**Review by cpanato — CHANGES_REQUESTED** [review]
thanks for the update!

see some small comments

**Comment by cpanato** [comment]
cc @puerco

**Comment by ba11b0y** [comment]
Thanks for the review @cpanato 
Pushed https://github.com/opensbom-generator/parsers/pull/58/commits/ab705f360acca3c9548fba5e41bf75c9a0b4112e

Also can you approve the workflow(first-time contributor here) so that I can test the PR
Ref: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -17,5 +17,14 @@ jobs:
         with:
           go-version: 1.19
           check-latest: true
+      - name: Setup node
+        uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c
```
Comment (cpanato) [inline .github/workflows/test.yml:21]: Please add a version in a comment in the same line. See the above for example

Thanks

**Review by cpanato — COMMENTED** [review]
thanks!

**Comment by cpanato** [comment]
need you here @puerco to approve the CI and PR

**Comment by nishakm** [comment]
@cpanato @ba11b0y I was expecting `go test -v` to run for  this PR, but I guess if it doesn't, I can test it on another PR. Otherwise LGTM!

**Pull request merged**